validation/azure/linux-vm/main.tf

```diff
@@ -16,15 +16,12 @@ module "linux_adextension" {
       version              = "test"
       caching              = "test"
       storage_account_type = "SSD"
-      disk_size_gb         = 128
+      disk_size_gb         = 64
```

Comment:
This is declared as an optional string. Why does this not fail?

Reply:
Number vs String in module, could be int32.

Reply:
Number vs String in module, could be int32.